infra/gcp/ensure-staging-storage.sh

```diff
@@ -52,6 +52,7 @@ STAGING_PROJECTS=(
     coredns
     csi
     descheduler
+    e2e-test-images
```

Comment:
We get one shot at naming.  Does "images" add enough value here, or is "e2e-test" or "test-e2e" or "testing/e2e" or even just "e2e" sufficient?